### ISSUE TITLE
Can work in the ui?

### ISSUE THREAD

**Issue by ShenKang4181** [yasirkula/UnityRuntimePreviewGenerator#10]
![image](https://user-images.githubusercontent.com/50401776/180595482-189159a1-c2e3-4625-a697-7ff1b1f7ef71.png)
I want to generate two preview images for these two objects.

I tried to run the following code
RuntimePreviewGenerator.GenerateModelPreview
But that return result is null

Can this plugin work within the ui?

**Comment by yasirkula** [comment]
You're right, it won't work. It's technically possible to render Screen Space-Camera and World Space canvases with a camera but the plugin checks for Renderer.bounds which doesn't apply to UI elements, at least not right now. Let's keep this Issue open and maybe in the future UI support can be added.
